src/main/java/frc/robot/RobotContainer.java

```diff
@@ -50,11 +54,14 @@ public class RobotContainer {
     private ArmSubsystem arm = new ArmSubsystem();
     private ShooterSubsystem shooter = new ShooterSubsystem();
     private IntakeSubsystem intake = new IntakeSubsystem();
+    private ClimbSubsystem climb = new ClimbSubsystem();
     private AutoBuilderSubsystem autoBuilder = new AutoBuilderSubsystem(positioning);
 
     // Commands
     private TeleopDriveCommand driveCommand = new TeleopDriveCommand(drive, positioning, this::getInput);
     private TeleopArmCommand armCommand = new TeleopArmCommand(arm, this::getInput);
+    private SequentialCommandGroup climbCommand =
+            new SequentialCommandGroup(new ZeroClimbersCommand(climb), new ClimbCommand(climb, this::getInput));
```

Comment:
If the climbers are already zeroed, we don't need to re-zero every time the robot is enabled. You should make the default command just the `ClimbCommand`, then use a trigger to re-zero on Autonomous or Teleop start, but only if the climber is not already zeroed.

Basically, duplicate the logic that already exists for the intake. See line 182 in this file.